examples/complete/main.tf

```diff
@@ -1,6 +1,8 @@
 
 locals {
-  name = "samplerepo"
+  name       = ["example-complete-repository", ]
+  partition  = data.aws_partition.current.partition
+  account_id = data.aws_caller_identity.current.account_id
```

Comment:
The untagged rule should be 1 day, by and large, you don't want untagged images, if they are there mist often than not it was by mistake or error and should be cleaned as soon as possible.

Reply:
This is going to sound weird but `30` images to keep make more sense ;)